plugins/source/pagerduty/codegen/recipes/base.go

```diff
@@ -45,6 +45,7 @@ type Resource struct {
 	ResponseStructOverride        string
 	ResponseFieldOverride         string
 	RestPathOverride              string
+	ParentIsPointer               bool
```

Comment:
I'm not a big fan of this... but since changing to something that doesn't require a field like this will be quite a lot of work, I'm willing to look the other way 😄